apps/condo/domains/user/schema/RegisterNewServiceUserService.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Generated by `createservice user.RegisterNewServiceUserService`
+ */
+
+const { GQLCustomSchema, getById } = require('@core/keystone/schema')
+const access = require('@condo/domains/user/access/RegisterNewServiceUserService')
+const { STAFF, MIN_PASSWORD_LENGTH } = require('@condo/domains/user/constants/common')
+const { MIN_PASSWORD_LENGTH_ERROR } = require('@condo/domains/user/constants/errors')
+const { ensureNotExists } = require('@condo/domains/user/utils/ensureNotExists')
+
+
+const RegisterNewServiceUserService = new GQLCustomSchema('RegisterNewServiceUserService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterNewServiceUserInput {  dv: Int!, sender: SenderFieldInput!, name: String!, email: String!, password: String! meta: JSON }',
```

Comment:
there is no need to send a password. it's better to create it randomly and return the result